app/components/accepted-record/template.hbs

```diff
@@ -0,0 +1,30 @@
+<table class="table table-filter">
+    <tbody>
+        {{#each model.acceptedPreprints.records as |record|}}
+            <tr data-status={{record.reviewsState}}>
+                <td class="state">
+                    <a class="fa fa-check-circle-o fa-lg"> </a>
+                </td>
+                <td>
+                    <div class="media">
+                        <div class="media-body" title={{record.title}}>
+                            <a href="/preprints/{{theme.id}}/{{record.id}}">
+                                <p class="record-title">{{shorten-text record.title screenWidth}}</p>
+                                <p class="record-body">
+                                    submitted on {{moment-format record.dateCreated "MMMM DD, YYYY"}} by
```

Comment:
We should be consistent with how the rest of the OSF displays datetimes. It looks closer to `YYYY-MM-DD H:M` (2017-01-13 01:18 PM). It's probably specified in ember OSF somewhere